pkgs/os-specific/linux/g933-utils/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, rustPlatform, udev, pkgconfig }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "g933-utils";
+  version = "0.13";
```

From the nixpkgs manual:
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 
```suggestion
  pname = "g933-utils-unstable";
  version = "2019-08.04";
```